app/components/QrCodeScanner/QrCodeScanner.jsx

```diff
@@ -0,0 +1,66 @@
+// @flow
+import React, { Component } from 'react'
+
+import Button from '../Button'
+import Instascan from 'instascan'
+
+import qrCodeScannerStyles from './QrCodeScanner.scss'
+
+type Props = {
+  callback: Function,
```

Comment:
Could you type the callback?
`callback: (content: Object) => any` (I'm unsure what is the type of content)

Reply:
updated